nixos/modules/system/boot/loader/systemd-boot/systemd-boot.nix

```diff
@@ -126,6 +126,19 @@ in {
         '';
       };
     };
+
+    graceful = mkOption {
+      default = false;
+
+      type = types.bool;
+
+      description = ''
+        Ignore errors during random seed operations. This can serve as a workaround
+        for systems on which the <literal>LoaderSystemToken</literal> EFI variable
+        is not writable, without disabling EFI variable access globally.
```

Comment:
What about something like

> Invoke `bootctl` with the `--graceful` option, which Ignores failure when the EFI System Partition cannot be found, or when EFI variables cannot be written. Currently only applies to random seed operations.

Maybe we should also add a warning that this can cause your system to be not bootable - Let's say if you missed adding the ESP, `bootctl` will stop complaining now.

Reply:
I didn't copy the man page verbatim, as I think it doesn't quite convey the practical implications/use of the configuration option.

> Maybe we should also add a warning that this can cause your system to be not bootable - Let's say if you missed adding the ESP, `bootctl` will stop complaining now.

That's what I was initially worried about as well, but upon closer inspection I believe this is not an issue at all:

    # bootctl install --esp-path=/tmp/test --graceful
    File system "/tmp/test" is not a FAT EFI System Partition (ESP) file system.

The changes (https://github.com/systemd/systemd/commit/351de38e4b4e6ca324346e6dbcefd224bbb3b190) are limited to *random operations only* - i.e. I don't believe they're relevant when *installing*. Systemd documentation is a bit vague in this regard because it states it's *currently* limited, suggesting this might change in the future.

As a result, the man page bit about the EFI system partition not being found seems irrelevant in our case as we're using it when installing only (it should fail regardless).

Another attempt:

> Invoke `bootctl install` with the `--graceful` option, which ignores errors when EFI variables related to random seed storage (e.g. *LoaderSystemToken*) cannot be written.

This:
- Is more explicit about applying to `bootctl install` only.
- A little less verbose on my particular use case
- Assumes *currently* will be the status quo for the foreseeable future

I have no strong objections against citing the manual verbatim if you (or anyone else) still feel that's the best approach.